CHANGELOG.md

```diff
@@ -1,5 +1,16 @@
 # Changelog
 
+## 14.0.0
+
+#### Breaking changes - [#525](https://github.com/openfisca/openfisca-core/pull/525)
+
+* Rename Variable attribute `url` to `reference`
+  - As previously, this attribute can be a string, or a list of strings.
+  - Variable attribute `law_reference` is removed.
+  - `TaxBenefitSystem.reference` renamed to `TaxBenefitSystem.reference_tbs`.
```

Comment:
Calling the original version of a variable in a reform “reference” doesn't sound very obvious to me. And I can't find a better alternative to “legislative reference”. Plus, if you consider how many times the current “reference” is used vs how many times “legislative reference” should be used (once for each variable), we should definitely prioritise the most common meaning.

I'm pretty sure we can find an acceptable alternate word for the “reference” tax benefit system in the context of a reform, which would solve both the `TaxBenefitSystem.reference` and `some_variable.reference` attribute in a reform.

What about the “base” tax benefit system? The “original”?

If none of these work out (please explain why), can we call the legislative reference `source`?